src/addressBook/index.test.js

```diff
@@ -247,27 +302,75 @@ describe("getPeople", () => {
       ]);
 
     jest
-      .spyOn(solidClientFns, "getStringNoLocale")
-      .mockReturnValueOnce(expectedPerson1.nickname)
-      .mockReturnValueOnce(expectedPerson1.name)
-      .mockReturnValueOnce(expectedPerson2.nickname)
-      .mockReturnValueOnce(expectedPerson2.name);
+      .spyOn(solidClientFns, "addStringNoLocale")
```

Comment:
Could we try to pass turtle files and let Solid client do its thing, instead of mocking the response from the various Solid Client methods?

Reply:
Alternatively, use the mocking methods that Solid client brings.

Reply:
Because of all the bugs we've found elsewhere, we'll defer changes to mocking.